stable/express-gateway/values.yaml

```diff
@@ -3,7 +3,7 @@
 
 image:
   repository: expressgateway/express-gateway
-  tag: v1.11.0
+  tag: v1.12.0
```

Comment:
please update readme to match as well

Reply:
and can you also sign off your commits? TIA

Reply:
@davidkarlsen Done!